samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -69,6 +69,15 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+Minimal Build
+=============
+
+The sample can be built with a minimum configuration as a demonstration of how to reduce code size and RAM usage.
```

Comment:
Prefere active voice:
You can build the sample with a minimum configuration...

Reply:
Done